### PR TITLE
Config: Add in docs property for 1.11.3

### PR DESCRIPTION
**Pull request by xbili**
This was preventing `grunt prepare` from running.

After looking through the code again from [this](https://github.com/jquery/download.jqueryui.com/pull/263), the if-else loop appears to be correct. My mistake on that. But the ```config.json``` file seems to require a ```doc``` property in order for the if-else loop to be executed correctly. 

If not ```npm install``` wouldn't run for ```tmp/api.jquery.com```. Do let me know if there are any mistakes with this! 

Fixes issue [262](https://github.com/jquery/download.jqueryui.com/issues/262) and also [this](https://github.com/jquery/jqueryui.com/issues/102).

Thanks!

### CODE REVIEW

**Comment by rxaviers** [comment]
Hi @xbili, thanks for your contribution.

Actually, 1.11 package (and on) doesn't embed documentation anymore. So, we shouldn't add `"docs": true` into the config. But, the prepare task is indeed broken (#262). In order to fix this, we actually should skip "Building API documentation for jQuery UI" if jqueryUi.docs is `undefined`.

**Comment by xbili** [comment]
I see, thanks for pointing that out. I've wrapped the process into an if-else loop in the ```prepare``` function instead. Do let me know if there are any issues with this. Thanks!

File: Gruntfile.js [hunk 1]
```diff
@@ -308,19 +308,24 @@ function prepare( jqueryUi ) {
...
-				}, log( callback, "Done building documentation", "Error building documentation" ) );
+					rimraf.sync( "tmp/api.jqueryui.com/dist" );
+					grunt.util.spawn({
+						cmd: "grunt",
+						args: [ "build" ],
+						opts: {
+							cwd: "tmp/api.jqueryui.com"
+						}
+					}, log( callback, "Done building documentation", "Error building documentation" ) );
+				}
```
Comment (rxaviers) [inline Gruntfile.js]: Please, I prefer you add a silent return callback and avoid the else condition (let else be the normal flow).
```js
if ( !jqueryUi.docs ) {
   return callback();
}
```

Comment (xbili) [inline Gruntfile.js]: @rxaviers Sorry about that, I've made the changes.

**Comment by rxaviers** [comment]
Thank you. Please, just let me know if you need this published for your `jqueryui.com` work.

**Comment by xbili** [comment]
@rxaviers You're welcome! I think manually running ```npm install``` for now will do. Thanks!

**Pull request closed without merging**